src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -415,15 +418,20 @@ public virtual AuthenticationProperties ConfigureExternalAuthenticationPropertie
         /// <summary>
         ///     Log SignInStatus for user and return SignInStatus
         /// </summary>
-        /// <param name="status"></param>
+        /// <param name="result"></param>
         /// <param name="user"></param>
         /// <param name="methodName"></param>
         /// <returns></returns>
-        protected async virtual Task<SignInResult> LogResultAsync(SignInResult status, TUser user, [System.Runtime.CompilerServices.CallerMemberName] string methodName = "")
+        protected virtual async Task<SignInResult> LogResultAsync(SignInResult result, TUser user,
+        [System.Runtime.CompilerServices.CallerMemberName] string methodName = "")
```

Comment:
Nit: Add a using with the namespace of the attribute everywhere and make this shorter.